shell/languages/nodejs.sh

```diff
@@ -7,15 +7,6 @@ source "$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)/../lib/she
 yarn_install_if_needed() {
   local stateFile="node_modules/devbase.lock"
 
-  if ! yarn -v >/dev/null 2>&1; then
```

Comment:
Duplicative code, we do this as part of `setup_environment`, which calls `ensure_asdf`, which is ran on local machines and CI.